Web/readme.md

```diff
@@ -1,13 +1,13 @@
 ![React](https://img.shields.io/badge/react-%2320232a.svg?style=for-the-badge&logo=react&logoColor=%2361DAFB)
 ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
 
-# Web
+# Web Client
```

Comment:
Why this file has been updated ? There are some errors such as Detox framework and react-native mobile mentions

Reply:
when Johan came to the branch and committed his readme file came too.